user_icons_loader.gd

```diff
@@ -0,0 +1,7 @@
+extends Node
+
+
+func _init() -> void:
```

Comment:
Icons will not work when you Export the game now, unless you ignore every folder and file except this file.